config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/ingredients", to: "ingredients#index"
+
+  get "/recipes/:id", to: "recipes#show"
+  patch "/recipes/:id", to: "recipes#update"
```

Comment:
Following the advice above, this route woudl change slightly; patching is for updating, so it's good your verb & action match, but consider again what the story is asking us to do: create a relationship.... What would change here?